app/src/main/java/com/xmartlabs/template/module/ControllerModule.kt

```diff
@@ -10,5 +10,5 @@ import javax.inject.Singleton
 class ControllerModule {
   @Provides
   @Singleton
-  internal fun provideSessionController() = SessionController(Session::class)
+  internal fun provideSessionController() = SessionController(Session::class.java)
```

Comment:
Kotlin has a `Class` wrapper called `KClass`. From kotlin, obtaining the class is via `::class`. But if you need the java instance, then you must use `::class.java`